docs/README.md

```diff
@@ -33,28 +33,28 @@ For more information, see ["Create a new billing account"](https://cloud.google.
 
 ## Deploy the Sandbox
 
-In a new browser tab, navigate to the Stackdriver Sandbox [website](https://stackdriver-sandbox.dev/) and follow the instructions there:
+In a new browser tab, navigate to the Cloud Operations Sandbox [website](https://stackdriver-sandbox.dev/) and follow the instructions there:
```

Comment:
Is there a plan to change the website URL (and content) to Cloud Ops Sandbox?

Reply:
yes, we will eventually change the URL too